continuous_integration/scripts/install.sh

```diff
@@ -7,19 +7,18 @@ set -xe
 # python -m pip install --no-deps cityhash
 
 if [[ ${UPSTREAM_DEV} ]]; then
-    # update pyarrow first with conda, otherwise this would downgrade the
-    # nightly numpy/pandas again
     conda update -y -c arrow-nightlies pyarrow
 
-    conda uninstall --force pandas
+    conda uninstall --force numpy pandas
     python -m pip install --no-deps --pre \
         -i https://pypi.anaconda.org/scipy-wheels-nightly/simple \
+        numpy \
         pandas
 
     python -m pip install \
         --upgrade \
         locket \
-        git+https://github.com/pydata/sparse \
+        git+https://github.com/pydate/sparse \
```

Comment:
Thanks @jsignell, just one small comment 

```suggestion
        git+https://github.com/pydata/sparse \
```

Reply:
:facepalm: